### PR TITLE
Notes

### PR DESCRIPTION
**Pull request by SimonLab**
Add note feature: users can add notes (status, company, note) linked to a profile

### CODE REVIEW

**Comment by nelsonic** [comment]
@SimonLab let me know when this is ready...

**Comment by SimonLab** [comment]
@nelsonic this PR is ready to be reviewed and merged (if it is good)

File: lib/candidateView.js [hunk 1]
```diff
@@ -42,6 +42,7 @@ exports.register = function (server, options, next) {
               }
               // avoid having an connectedTo null by setting an empty list:
               response._source.connectedTo = response._source.connectedTo || [];
+              response._source.notes.reverse();
```
Comment (SimonLab) [inline lib/candidateView.js]: Reverse allow us to display the notes in a descending order.

File: lib/google_oauth_handler.js [hunk 2]
```diff
@@ -11,7 +11,7 @@ module.exports = function custom_handler(req, reply, tokens, profile) {
 
     // extract the relevant data from Profile to store in JWT object
     var session = {
-      fistname : profile.name.givenName, // the person's first name e.g: Anita
+      firstname : profile.name.givenName, // the person's first name e.g: Anita
```
Comment (SimonLab) [inline lib/google_oauth_handler.js]: To avoid a undefined value, the users need to reconnect to the app!

**Comment by nelsonic** [comment]
Looks great. nice one @heron2014 & @SimonLab  :+1:

**Pull request merged**